presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -1190,6 +1190,15 @@ The table is partitioned by the transformed value of the column::
 
      ALTER TABLE iceberg.web.page_views ADD COLUMN ts timestamp WITH (partitioning = 'hour');
 
+ALTER VIEW
+^^^^^^^^^^
+
+Alter view operations to alter the name of an existing view to a new name is supported in the Iceberg connector::
```

Comment:
```suggestion
Alter view operations to alter the name of an existing view to a new name is supported in the Iceberg connector.
```

Using `::` here causes the code block to look like this in a local doc build: 
<img width="984" alt="Screenshot 2024-11-13 at 10 44 22 AM" src="https://github.com/user-attachments/assets/481d3e50-ecac-4155-92ed-8c41ff895a2b">

I tested this edit in a local doc build and it fixes the code block.